utilities/shared_disk.py

```
@@ -0,0 +1,144 @@
+    vm2_user, vm2_pass = get_ssh_credentials_from_provider_config(source_provider_data, vm2_info)
+
+    ssh_vm1 = vm_ssh_connections.create(vm_name=vm1_name, username=vm1_user, password=vm1_pass)
+    ssh_vm2 = vm_ssh_connections.create(vm_name=vm2_name, username=vm2_user, password=vm2_pass)
+
+    mount_point = "/mnt/shared_disk"
+    partition = f"{shared_disk_device}1"
+    test_file_vm1 = f"{mount_point}/test-vm1.txt"
+    test_file_vm2 = f"{mount_point}/test-vm2.txt"
+
+    # VM1: Mount shared disk and write test data
+    LOGGER.info(f"VM1 ({vm1_name}): Mounting shared disk {partition}")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)